lib/hashie/extensions/coercion.rb

```diff
@@ -178,7 +178,7 @@ def value_coercion(value)
         def inherited(klass)
           super
 
-          klass.key_coercions = key_coercions
+          klass.key_coercions = key_coercions.dup
```

Comment:
So this is a fix around superclass pollution, this definitely needs a CHANGELOG entry please.